packages/cli/src/commands/grandamento/get-buy-amount.ts

```diff
@@ -0,0 +1,55 @@
+import { StableToken } from '@celo/contractkit'
+import { toFixed } from '@celo/utils/lib/fixidity'
+import { flags } from '@oclif/command'
+import { BaseCommand } from '../../base'
+import { printValueMap } from '../../utils/cli'
+import { Flags } from '../../utils/command'
+import { enumEntriesDupWithLowercase } from '../../utils/helpers'
+
+const stableTokenOptions = enumEntriesDupWithLowercase(Object.entries(StableToken))
+
+export default class GetBuyAmount extends BaseCommand {
+  static description = 'Gets the buy amount for a prospective Granda Mento exchange'
+
+  static flags = {
+    ...BaseCommand.flags,
```

Comment:
Testing is failing with ` --from FROM  Account Address`
```suggestion
```

Reply:
Battled tests for like 2 days straight- that was a bit of a red herring, that particular log was from a different test. Eventually got all the tests working though!